engine/src/test/java/org/camunda/bpm/engine/test/standalone/db/SchemaLogEnsureSqlScriptTest.java

```diff
@@ -55,7 +55,11 @@ public void ensureUpgradeScriptsUpdateSchemaLogVersion() {
         scriptsForDB.add(file);
       }
     }
-    assertThat(getLatestTargetVersion(scriptsForDB), is(currentSchemaVersion));
+
+    // exclude databases that are newly added
+    if (!scriptsForDB.isEmpty()) {
+      assertThat(getLatestTargetVersion(scriptsForDB), is(currentSchemaVersion));
+    }
```

Comment:
By this, `careful: the .0 must be hardcoded or else it will break as soon as we build a first patch)`, I assume you meant if we build a (first) patch that doesn't contain an upgrade script, as the Schema log entry will remain on the previous (minor) version.

I solved this by getting the minor version from the projects `pom.xml`, through a `*.properties` file, patch versions are removed, and a `.0` is appended.